pyomo/gdp/plugins/bigm.py

```diff
@@ -507,7 +521,8 @@ def _xform_constraint(self, obj, disjunct, infodict,
         # Though rare, it is possible to get naming conflicts here
         # since constraints from all blocks are getting moved onto the
         # same block. So we get a unique name
```

Comment:
With @emma58 changes to Big-M, is this stipulation still true? 

> Though rare, it is possible to get naming conflicts here
> since constraints from all blocks are getting moved onto the
> same block. So we get a unique name